app/views/core/CocoView.coffee

```diff
@@ -246,11 +246,10 @@ module.exports = class CocoView extends Backbone.View
     if me.isStudent()
       console.error("Student clicked contact modal.")
       return
+
     if me.isTeacher(true)
       if application.isProduction()
-        window.Intercom?('show')
-      else
-        alert('Teachers, Intercom widget only available in production.')
+        application.tracker.drift.sidebar.open()
```

Comment:
Not able to backtrack this, is this calling `get drift ()` from `Tracker2/index.js`, and then eventually referring to `window.drift`?

Reply:
or maybe `driftApi`?

Reply:
This is calling `get drift()`  from `Tracker2`, application.tracker is set in application.coffee during application initialization.  Any thoughts on how I can make this more clear?

Reply:
That's fine, I think I was missing context from other PRs.